pkg/apis/core/types.go

```diff
@@ -1966,6 +1966,33 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
+// ContainerResizePolicy specifies user guidance on how container resource resize should be handled.
+// Only one of the following container resize policies may be specified.
+// If none of the following policies is specified, it defaults to NoRestart.
```

Comment:
The remarks about default behavior are out of place here, and should go on the field not the type.

Reply:
Also, is that the safe default?

Reply:
By safe, do you mean 'wont have any undesirable results'?

Reply:
Assume 90% of users use the default.  Are we going to behave correctly for most of them most of the time?  It seems like "Restart" is the more conservative default, and "NoRestart" is more idealistic.